src/events/WebComponentCustomEvents.js

```diff
@@ -15,3 +15,9 @@ export const runStartedEvent = webComponentCustomEvent("editor-runStarted");
 
 export const stepChangedEvent = (detail) =>
   webComponentCustomEvent("editor-stepChanged", detail);
+
+export const logInEvent = (detail) =>
+  webComponentCustomEvent("editor-logIn", detail);
+
+export const signUpEvent = (detail) =>
+  webComponentCustomEvent("editor-signUp", detail);
```

Comment:
You only need to add the detail if we're passing extra data to the event (see the `codeChangedEvent` for instance), but also if you want to keep it like this for consistency I don't mind...

Reply:
Removed!

Reply:
On this, I guess there are two options for consistency here: either make the events with no detail all events or all functions that just return the event... 🤔 Either way I guess we'll need to change how some of them are referenced (either `myCustomEvent` or `myCustomEvent` depending on which we go with)